README.md

```diff
@@ -150,10 +148,68 @@ let scaleString = scale.stringValue
 let scaleInt = scale.integerValue
 ```
 
+#### Type Declarations ####
+
+When declaring types, the colon should be placed immediately after the identifier followed by one space
+and the type name.
+
+```swift
+
+var intValue: Int
+
+// Do NOT do this
+var intValue : Int
+
+```
+
+In all use-cases, the colon should be associated with the left-most item with no spaces preceding and one space afterwards:
+
+```swift
+let myDictionary: [String: AnyObject] = ["String": 0]
+```
+
+#### typealias ####
+
+Typealias declaration should precede any other type declaration.
+
+
+```swift
+
+typealias ClosureType = (ParameterTypes) -> (ReturnType)
+typealias AgeAndNameProcessor = (Integer, String) -> Void
+
+var intValue: Int
```

Comment:
I would put a class or a struct declaration as an example instead, so it's clear that `typealias` is outside of a type scope